packages/metadata-crawler/src/crawler.ts

```diff
@@ -199,6 +209,39 @@ async function processAttestationServiceStatusForValidator(
     },
     'checked_attestation_service_status'
   )
+
+  // Consider pushing a state change to Discord for elected validators.
+  if (discordWebhook && isElected) {
+    const currentValid =
+      status.state === AttestationServiceStatusState.NoAttestationSigner ||
+      status.state === AttestationServiceStatusState.UnreachableHealthz ||
+      status.state === AttestationServiceStatusState.Valid
+
+    if (!currentValid) {
+      const content =
+        `:no_mobile_phones: **Problem with Attestation Service!** ${status.name} \n` +
+        `For validator \`${validator.address}\` in group \`${validator.affiliation}\`\n` +
```

Comment:
Consider adding the name here

Reply:
+1 to adding Validator name and group name if available -- sometimes the validator name doesn't make it immediately obvious which group it belongs to

Reply:
We already display the validator name with `${status.name}`.  Group name is a bit harder.

Reply:
@nambrot are we getting discord name claims soon? would love to @ mention the handle in the metadata.